build/Makefile

```diff
@@ -7,6 +7,7 @@
 
 # Set up architecture and Build Root Directory
 # PF (i.e. PlatForm) is either linux, solaris
+SHELL=/bin/bash
```

Comment:
I'm not sure this is a good change. In fact, I believe it's not.

Is this a ULINUX-specific change? If so, should it go in Makefile.pf.Linux?

My concern is about UNIX systems (AIX, HP, Solaris). Does /bin/bash exist on all platforms there? Is this a valid change (right now, it's not technically required for purposes) on UNIX platforms?